### ISSUE TITLE
Rumble support

### ISSUE THREAD

**Issue by Epacik** [FeralAI/GP2040#38]
Rumble support could be a nice addition to a controller made using this project. 
Especially since, at least in pi pico, there are some unused pins.

**Comment by FeralAI** [comment]
Agreed this would be a nice feature, and all the RP2040 pins can do PWM. Will need to do some research around this (USB descriptor handling, validating load on RP2040 pins, etc.), but will add it to the feature project list.

**Comment by Infer4Y** [comment]
I was looking at https://github.com/dmadison/ArduinoXInput/blob/master/src/XInput.h#L54-L57
https://github.com/dmadison/ArduinoXInput/blob/master/src/XInput.cpp#L466-L469

I hope this might help?
